Added forecast category criteria (#11309)
PostHog/posthog.com: contents/handbook/growth/sales/crm.md

6. Closed Won (100%) - They have signed the contract and are officially a PostHog customer.
7. Closed Lost (0%) - At some point in the pipeline they decided not to use us. The Loss Reason field is required for any opportunity to be marked as Closed lost.

### Forecast categories

> **Commit:** PostHog is integrated and the buyer has stated an intent to purchase within the Close Date quarter

Reviewer: ```suggestion
**Commit:** PostHog is integrated and the buyer has stated an intent to purchase within the Close Date quarter.
```

> **Best Case:** PostHog is or is being implemented, volume justifies an annual commitment, and the buyer has stated interest in purchaseing with the Close Date quarter

Reviewer: ```suggestion
**Best Case:** PostHog is or is being implemented, volume justifies an annual commitment, and the buyer has stated interest in purchasing with the Close Date quarter.
```

**Pipeline:** Buyer is actively evaluating PostHog or intends to evaluate PostHog within the Close Date quarter and early volume/discussion indicates an annual contract could be justified.

> **Ommitted:** Use this for new Opportunities that have have not met the criteria above

Author: Let's leave it out then? I think the other criteria apply.

> **Ommitted:** Use this for new Opportunities that have have not met the criteria above

Reviewer: Omitted = Closed Lost?  or move to the next quarter?  Maybe both actually.

> **Ommitted:** Use this for new Opportunities that have have not met the criteria above

Reviewer 2: Yep, you can omit by removing from the quarter (and therefore the forecast) or closing it.

> **Ommitted:** Not used. You can omit from Forecast by moving the Opportunity to a new quarter or closing it.

Reviewer: ```suggestion
**Omitted:** Not used. You can omit from Forecast by moving the Opportunity to a new quarter or marking it as Closed - Lost.
```
